Bembel/src/Geometry/SurfacePoint.hpp

```diff
@@ -40,4 +41,13 @@
  **/
 typedef Eigen::Matrix<double, 12, 1> SurfacePoint;
 
+/**
+ * \ingroup Geometry
+ * \brief typedef std::vector<SurfacePoint> with aligned allocator of Eigen for
+ * compatibility with older compilers.
+ */
+typedef std::vector<SurfacePoint, Eigen::aligned_allocator<SurfacePoint>>
```

Comment:
This is just for compatibility with older compilers?

Reply:
Yes, see [https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html](https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html)